extensions/2.0/Khronos/KHR_materials_variants/README.md

```diff
@@ -0,0 +1,151 @@
+# KHR_materials_variants
+
+## Contributors
+
+- Mike Bond, Adobe, [@Miibond](https://github.com/MiiBond)
+- Aurélien Vaysset, Emersya, [@aurelienvaysset](https://github.com/aurelienvaysset)
+- Pär Winzell, Facebook, [@zellski](https://twitter.com/zellski)
+- Renee Rashid, Facebook, [@debuggrl](https://github.com/debuggrl)
+- Jeremy Cytryn, Facebook, [@jercytrn](https://github.com/jercytryn)
+- Edgar Simson, OLV, [@Edzis](https://github.com/edzis)
+- Daniel Beauchamp, Shopify, [@pushmatrix](http://github.com/pushmatrix)
+- Stephan Leroux, Shopify, [@sleroux](http://github.com/sleroux)
+- Eric Chadwick, Wayfair, [@echadwick-wayfair](https://github.com/echadwick-wayfair)
+
+Copyright (C) 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows for a compact glTF representation of multiple material variants of an asset, structured to allow low-latency switching at runtime.
+
+A typical use case is digital commerce, where a user might be presented with e.g. a pair of sneakers and the ability to switch between different colours.
+
+![pair of sneakers](examples/shoes/photo.png)
```

Comment:
Oops, this became a broken link.  Maybe add this image back alongside README.md (not in a folder)?